app/views/welcome/categories.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="col-sm-4">
+    <div class="content-container">
+      <%= image_tag('categories/original.jpg', :alt => 'original art category photo', :class => 'categoryImage') %>
+      <p class="categoryLabel">ORIGINAL ART</p>
+    </div>
+  </div>
+  <div class="col-sm-4">
+    <div class="content-container">
+      <%= image_tag('categories/prints.jpg', :alt => 'print art category photo', :class => 'categoryImage') %>
+      <p class="categoryLabel">PRINTS</p>
+    </div>
+  </div>
+  <div class="col-sm-4">
+    <div class="content-container">
+      <%= image_tag('categories/stationery.jpg', :alt => 'stationery category photo', :class => 'categoryImage') %>
+      <p class="categoryLabel">STATIONERY</p>
+    </div>
+  </div>
+  <div class="col-sm-4">
+    <div class="content-container">
+      <%= image_tag('categories/jewelry.jpg', :alt => 'jewelry category photo', :class => 'categoryImage') %>
+      <p class="categoryLabel">JEWELRY</p>
+    </div>
+  </div>
+  <div class="col-sm-4">
+    <div class="content-container">
+      <%= image_tag('categories/wearable.jpg', :alt => 'wearables category photo', :class => 'categoryImage') %>
+      <p class="categoryLabel">WEARABLES</p>
```

Comment:
this should be category-label
make sure you are following naming conventions and best practices for the languages